WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -607,6 +648,47 @@ public void onDismissed(Snackbar snackbar, int event) {
         snackbar.show();
     }
 
+    /**
+     * Searches the PostStore for post with a matching remote ID. Used when searching since PostModel's returned
+     * from search actions don't have valid local IDs
+     */
+    private PostModel getPostFromRemoteId(long remoteId) {
```

Comment:
This should be done in FluxC. I'd suggest removing this completely and sending a PR against FluxC which adds `PostStore.getPostByRemotePostId()` and using that here instead.